tests/testthat/test-ictr.R

```diff
@@ -23,3 +23,21 @@ test_that("outputs the expected columns", {
   out <- ictr(companies, inputs)
   expect_false(dplyr::is_grouped_df(out))
 })
+
+test_that("if a company matches no inputs, all shares are `NA` (#176)", {
+  companies <- tibble(
+    company_id = "a",
+    activity_uuid_product_uuid = "a"
+  )
+  inputs <- tibble(
+    activity_uuid_product_uuid = "b",
+    input_co2 = 1,
+    input_sector = "transport",
+    unit = "metric ton*km",
+  )
+
+  out <- ictr(companies, inputs)
+
+  all_is_na <- all(is.na(unlist(select(out, starts_with("score")))))
+  expect_true(all_is_na)
```

Comment:
This currently fails. Fix the code until this test passes -- ensuring all other tests also pass unless this new requirement conflicts with some previous requirement in which case we'll need to discuss to untangle the conflict.

Reply:
@maurolepore I have updated an old test to better suit our current understanding of the final output. Please review the changes

Reply:
Thanks Kalash. It might be easier to read if we split different cases in different tests (one case with a company matching no product, another case with a company matching some produce).

I may follow  up in another PR.